_includes/responsive-image.html

```diff
@@ -0,0 +1,58 @@
+{% comment %}
+<!--
+  Responsive Image Include
+
+  This partial is used with the responsive image plugin.  In the _config file,
+  this template is used as the default for responsive images to configure a
+  srcset value for images to ensure the best option for an image is used for a
+  specific user's browser.
+
+  @param {string} "path" - Required. Path to image file relative to the root
+  @param {string} "alt" - Required. Alt tag for image
+  @param {boolean} "img_link" - Optional, default is false. If img should have
+    popover link
+  @param {boolean} "fig" - Optional, default is false. If image should be
+    inside a figure tag
+  @param {string} "fig_caption" - Optional. Caption to be used with figure tag
+  @param {string} "rendition" - Optional.  If image should have sizes options.
```

Comment:
It's a `string` type, but the description sounds like a bool. What are the possible values for this? Also, is it `rendition` or `media_rendition`? It looks like it's getting called as `media_rendition`.